src/subscription_manager/cli_command/register.py

```diff
@@ -233,9 +235,14 @@ def _do_command(self):
                     type=self.options.consumertype,
                     service_level=self.options.service_level,
                 )
-        except (connection.RestlibException, exceptions.ServiceError) as re:
```

Comment:
hm why not simply add `ServiceError` to the `ExceptionMapper`? this why there is no separate handling, and it is easier to change it in the future. IMHO a simple implementation that calls `str(e)` should do it

Reply:
I've done some changes, see the comment and the second commit.